docs/source/concepts.rst

```diff
@@ -116,9 +130,66 @@ completed iterations and epochs:
 
     trainer.add_event_handler(Events.ITERATION_COMPLETED, on_iteration_completed)
 
+There are many ways of creating a `process_function`, this is why Ignite provides output_transform parameter for its 
```

Comment:
I would rephrase this as 
```suggestion
Since there is no restrictions on `process_function`, Ignite provides `output_transform` argument for its 
```
@anmolsjoshi what do you think ?

Reply:
It reads better!